src/wallet/walletdb.cpp

```diff
@@ -808,13 +808,14 @@ struct MWCoin {
         READWRITE(fKey);
         if (fKey) {
             if (ser_action.ForRead()) {
-                libmw::PrivateKey key;
-                READWRITE(key.bip32Path);
-                READWRITE(key.keyBytes);
-                coin.key = boost::make_optional(std::move(key));
+                libmw::BlindingFactor bf;
+                READWRITE(bf);
+                coin.key = boost::make_optional(std::move(bf));
```

Comment:
Currently, the existence of key also implies the existence of blind, and vise versa. But I don't know that we should always make that assumption. Let's add another bool here for checking if blind is available. So something like this:

```
        bool fKey = coin.key != nullopt;
        READWRITE(fKey);
        if (fKey) {
            if (ser_action.ForRead()) {
                libmw::BlindingFactor key;
                READWRITE(key);
                coin.key = boost::make_optional(std::move(key));
            } else {
                READWRITE(*coin.key);
            }
        }

        bool fBlind = coin.blind != nullopt;
        READWRITE(fBlind);
        if (fBlind) {
            if (ser_action.ForRead()) {
                libmw::BlindingFactor blind;
                READWRITE(blind);
                coin.blind = boost::make_optional(std::move(blind));
            } else {
                READWRITE(*coin.blind);
            }
        }
```